pkg/configs/api/api.go

```diff
@@ -19,15 +20,38 @@ import (
 	"github.com/cortexproject/cortex/pkg/util"
 )
 
+// Config configures Configs API
+type Config struct {
```

Comment:
Is this the right place where it should stay? It both allow to configure DB and API, so given it's the root block I would see it more correctly places in `pkg/configs`. The clashing `type Config` in `pkg/configs` could be more correctly renamed `UserConfig`, given it contains the configuration submitted by a single user? What's your take?